### DISCUSSION TITLE
Welcome to QUICK Discussions!

### DISCUSSION THREAD

**Discussion by merzlab** [merzlab/QUICK#239] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jimkress65** [comment]
How can MOLDEN and NBO files be generated using QUICK?

**Reply by agoetz** [reply]
MOLDEN files can be written by adding the keyword `MOLDEN` to the input file. There is currently no NBO support.
